src/SitemapServiceProvider.php

```diff
@@ -18,10 +19,19 @@ public function boot()
             __DIR__.'/../resources/views' => base_path('resources/views/vendor/laravel-sitemap'),
         ], 'views');
 
+        $this->publishes([
+            __DIR__.'/../resources/config/laravel-sitemap.php' => config_path('laravel-sitemap.php'),
+        ], 'config');
+
         $this->app->when(SitemapGenerator::class)
             ->needs(Crawler::class)
             ->give(function () {
-                return Crawler::create();
+                return Crawler::create([
+                    RequestOptions::COOKIES => config('laravel-sitemap.cookies'),
```

Comment:
instead of passing options individually here, pas there entire `config('sitemap.guzzle_options)` here.